boards/infineon/kit_psc3m5_evk/doc/index.rst

```diff
@@ -0,0 +1,122 @@
+.. zephyr:board:: kit_psc3m5_evk
+
+Overview
+********
+
+This is the standard evaluation board for our PSOC™ Control C3 family of MCU's. PSOC™ Control C3M5
```

Comment:
"our" should be avoided please. This is Zephyr documentation, not Infineon's :)
Please review/fix this kind of wording in the entire file